wgsl/index.bs

```diff
@@ -12837,7 +12838,7 @@ specify the component type; the component type is inferred from the constructor
       <td>
         <xmp highlight=wgsl>@const @must_use fn u32(e : T) -> u32</xmp>
   <tr><td>Parameterization
-      <td>`T` is a [=type/concrete=] [=scalar=] type or [=AbstractInt=]
```

Comment:
Similar comment here as above.
We can test the preservation of all 32 mantissa bits.